catalog/spatial/kml/spatial-kml-transformer/src/main/java/org/codice/ddf/spatial/kml/transformer/HandlebarsMetacard.java

```diff
@@ -78,6 +78,19 @@ public AttributeFormat setValue(AttributeFormat value) {
         public int compareTo(AttributeEntry other) {
             return this.getKey().getName().compareTo(other.getKey().getName());
         }
+        @Override
+        public boolean equals(Object other) {
+            if (other != null && other instanceof AttributeEntry) {
+                AttributeEntry attributeEntry = (AttributeEntry)other;
+                return this.getKey().getName().equals(attributeEntry.getKey().getName());
+            }
+            return false;
+        }
+
+        @Override
+        public int hashCode() {
+            return this.getKey().getName().hashCode();
```

Comment:
`this.getKey()` seems overkill here. How about using `key` directly?

Also, any chance `key` or `key.getName()` could be `null`? Same question for the `equals()` method above.

We may want to consider using IntelliJ's _Generate equals() and hashCode()_ action.